opsdroid/connector/rocketchat/__init__.py

```diff
@@ -0,0 +1,190 @@
+"""A connector for Rocket.Chat."""
+import asyncio
+import logging
+import aiohttp
+
+from opsdroid.connector import Connector
+from opsdroid.message import Message
+
+_LOGGER = logging.getLogger(__name__)
+API_PATH = '/api/v1/'
+
+
+class RocketChat(Connector):
+    """A connector for the chat service Rocket.Chat."""
+
+    def __init__(self, config):
+        """Create the connector.
+
+        Sets up logic for the Connector class, gets data from
+            the config.yaml or adds default values.
+
+        Args:
+            config (dict): configuration settings from the
+                file config.yaml.
+
+        """
+        super().__init__(config)
+        self.name = "rocket.chat"
+        self.config = config
+        self.default_room = config.get("default-room", "general")
+        self.group = config.get("group", None)
+        self.url = config.get("channel-url", "https://open.rocket.chat")
+        self.update_interval = config.get("update-interval", 1)
+        self.bot_name = config.get("bot-name", "opsdroid")
+        self.listening = True
+        self.latest_update = None
```

Comment:
Should we instantiate this to now? That way if you restart opsdroid it will not respond to historic messages.

Reply:
How should this be done? Just to check how to best alter this

Reply:
Using `datetime.datetime.now().strftime('some formatting string')`. You'll need to find the right formatting string in the docs to match the format used by the API (looks like full ISO 8601 to me).